app/src/main/java/org/jellyfin/androidtv/ui/playback/MediaManager.java

```diff
@@ -986,7 +986,9 @@ public void resumeAudio() {
     }
 
     public void setCurrentMediaPosition(int currentMediaPosition) {
-        this.mCurrentMediaPosition = currentMediaPosition;
+        if (!hasVideoQueueItems() || currentMediaPosition < 0 || currentMediaPosition > getCurrentVideoQueue().size())
```

Comment:
This breaks the old and new photo viewers (app crashes when trying to view a photo)